lib/licensed/sources/bundler.rb

```diff
@@ -135,9 +166,23 @@ def bundle_exec_gem_spec(name)
         # `gem` must be available to run `gem specification`
         return unless Licensed::Shell.tool_available?("gem")
 
-        # use `gem specification` with a clean ENV to get gem specification YAML
-        yaml = ::Bundler.with_original_env { Licensed::Shell.execute(*ruby_command_args("gem", "specification", name)) }
-        Gem::Specification.from_yaml(yaml)
+        # use `gem specification` with a clean ENV and clean Gem.dir paths
+        # to get gem specification at the right directory
+        begin
+          ::Bundler.with_original_env do
+            ::Bundler.rubygems.clear_paths
```

Comment:
Another day, another bundler fix.  After surfacing errors for specification paths that don't exist on disk, I started seeing errors running `bundler exec licensed -c test/fixtures/config/.licensed.yml` for this repo.  The error was for the bundler gem, which was trying to be found under the `vendor/gems` folder.

Clearing the ruby gems paths inside of `with_original_env` seemed to fix it so that it correctly picked up the global gem folder as the root of the specification's `gem_dir`